src/cmd/logic-function.test.js

```diff
@@ -329,21 +340,22 @@ describe('LogicFunctionCommands', () => {
 
 			const paths = ['dir/', 'dir/path/to/file'];
 
-			const res = await logicFunctionCommands._validatePaths({ paths });
+			const res = await logicFunctionCommands._validatePaths({ dirPath: paths[0], jsonPath: paths[1], _exit: false });
```

Comment:
Feels too odd to have an `_exit` property that is defined as an `fn` and here you have it as `boolean`

Reply:
@hugomontero  can u please elaborate on what feels odd about this?

Reply:
By default you are telling in the definition that `exit` is a `function` but over here you are passing `exit` as a `boolean` so you are breaking with the type definition, in case the code reach that `exit` that now is `false` it will throw you an error indicating that `exit` is not a function

Reply:
This is an example of what is happening with this code:
![image](https://github.com/particle-iot/particle-cli/assets/6526979/0aab0d24-2b2e-4d22-a3d4-3e93735303a1)

Reply:
this is great. thanks for sharing